napari/_qt/menus/_tests/test_plugins_menu.py

```diff
@@ -30,3 +30,21 @@ def widget2():
     plugin_action_menu.actions()[0].trigger()
     assert len(viewer.window._dock_widgets) == 1
     assert list(viewer.window._dock_widgets.data)[0] == 'Widget 1 (tmp_plugin)'
+
+
+def test_plugin_manager(make_napari_viewer):
+    """Test that the plugin manager is accessible from the viewer"""
+    viewer = make_napari_viewer()
+
+    plugins_menu = viewer.window.plugins_menu
+    assert plugins_menu._plugin_manager_dialog_cls is not None
+
+    actions = plugins_menu.actions()
+    for action in actions:
+        if action.text() == "Plugin Manager":
+            break
+    else:
```

Comment:
```suggestion
    else:  # pragma: no cover
```

as we do not expect to trigger this in normal test run